pkg/k8s/deployments/translate.go

```diff
@@ -99,6 +99,10 @@ func translate(t *model.Translation, ns *apiv1.Namespace, c *kubernetes.Clientse
 	TranslatePodAffinity(&t.Deployment.Spec.Template.Spec, t.Name)
 	for _, rule := range t.Rules {
 		devContainer := GetDevContainer(&t.Deployment.Spec.Template.Spec, rule.Container)
```

Comment:
should `GetDevContainer` return an error instead?

Reply:
i think it's fair for it to return nil, and the caller to deal with it.